test/test_main.cpp

```diff
@@ -79,6 +77,31 @@ int main(int argc, char* argv[])
         return returnCode;
     }
 
+        // If we're talking about a particular OpenCL device,
```

Comment:
Indentation...?

Reply:
`make format` screws it up, this way. However, the comment is really only even relevant to `benchmarks`. Here, the global variable isn't used at all unless OpenCL is enabled and being tested. So, this preprocessor `#else` branch can just be totally removed, without any loss of clarity.